lsp-clients.el

```diff
@@ -511,6 +511,30 @@ responsiveness at the cost of possibile stability issues."
                   :add-on? t
                   :server-id 'angular-ls))
 
+;;; C-sharp
+(defgroup lsp-csharp nil
+  "LSP support for C#, using the Omnisharp Language Server. Version 1.34.3 minimum required."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/OmniSharp/omnisharp-roslyn"))
+
+(defcustom lsp-clients-csharp-language-server-path
+  (expand-file-name "~/.omnisharp/omnisharp/omnisharp/OmniSharp.exe")
```

Comment:
is 3 times omnisharp intentional?

Reply:
If you unpack the OmniSharp Roslyn tarball in ‘~/.omnisharp/‘, this is what you get, yes.